06-building-a-program-start-to-finish/solving-the-problem/output/invertindex.js

```diff
@@ -0,0 +1,216 @@
+/*eslint-disable */
```

Comment:
Ok thanks, i just discovered a more better way to do it using "Singleton in ES6"